ADAPTERS.md

```diff
@@ -1,28 +1,101 @@
+
+This repo accesses adapter-specific logic in [adapterIndex.ts](./apps/server/src/adapterIndex.ts)
+
+## Test institutions
+
+Adding a test institution to the [default institution list](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) allows you to test your adapter. 
+
+For now, add this manually to the [ucwInstitutionsMapping.json](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) file. In the future, UCP will have a way to manage and sync your aggregator and aggregator/institution mappings via an online service.
+
+```typescript
+const testInstitution = {
+  "name": "My Test Bank",
+  "id": "00000000-0000-0000-0000-000000000000",
+  "keywords": ["test bank", "text bank", "test bakn"],
+  "logo": "https://universalconnectproject.org/images/ucp-logo-icon.svg",
+  "url": "https://my-test-bank.com",
+  "is_test_bank": true,
+  "routing_numbers": [],
+  "myAggregatorName": {
+    "id": "myAggregatorId",
+    "supports_aggregation": true,
+    "supports_history": true,
+    "supports_identification": true,
+    "supports_oauth": true,
+    "supports_verification": true
+  }
+};
+```
+## Monorepo 
+
+Because this repo is a monorepo, here are some caveats to consider with regard to adapter creation.
+
+### Aggregator adapter package and Turborepo
+
+There are two ways to create a shared package within a Turborepo monorepo. You can create Just-in-Time (JIT) packages or compiled packages. Please see [Turborepo's Documentation](https://turbo.build/repo/docs/core-concepts/internal-packages#compiled-packages) for more detailed information.
+
+Adapter Packages are considered compiled packages, and even though end-users will install your adapter package using NPM, the preferred way what we've chosen to create and test an adapter package is to use it as a compiled package.
+
+Below you'll find some tips on how to successfully run the UCW with a compiled package.
+
+We will use [the MX Adapter](https://github.com/Universal-Connect-Project/ucw-adapter-mx/tree/main/packages/mx-adapter) as a reference.
+
+#### Development tips
+
+When developing an adapter package, you'll need to install it in your monorepo workspace in order to test it. You can do this using the following command:
+
+```bash
+npm install <your_package_name> --workspace apps/server
+```
+_Package name is the `name` property in your adapter package's `package.json`_
+
+Once you run the above command, change the version of the package to a start with `*`. In the context of npm, this means install the latest version of the package. This is also a convention of Turborepo, and is what is recommended when developing package locally. 
```

Comment:
Are you sure this works? The turborepo internal package documentation says to use @repo/ for internal packages. This feels like it's running off a previously installed version of the package that was added during the npm install command.

Reply:
@wesrisenmay-mx `@repo/` is just a convention that they picked because there aren't any packages on npm with the `@repo` scope. @repo doesn't really have any special meaning in the turborepo context. 

This screenshot is taken from the turborepo docs: 
<img width="833" alt="image" src="https://github.com/user-attachments/assets/5a1d54d3-3de1-4239-93e9-3d7f8948a583">

I've tested this quite extensively. If the output directory exists (in this case, `dist`), and the proper `exports` are set up in the package file, then the project will use the local assets, even if the package exists in the npmjs repository.

Reply:
@wesrisenmay-mx I added the additional step.